app/services/users.js

```diff
@@ -3,3 +3,6 @@ const { User } = require('../models');
 exports.createUser = userParams => User.create(userParams);
 
 exports.findUserByEmail = email => User.findOne({ where: { email } });
+
+exports.findAndCountAllUsers = ({ offset, limit }) =>
+  User.findAndCountAll({ offset, limit, order: [['id', 'asc']] });
```

Comment:
Ordering is not necessary. The card does not asked for it.

Reply:
But without a default order, rows could be retrieved on different order when iterating throught pages